### ISSUE TITLE
Need non-versioned biblio references

### ISSUE THREAD

**Issue by moekraft** [tabatkins/bikeshed#1359]
In the ACT Rules Format, we use the Bikeshed reference tags to pull in Normative and Informative references. We would like to refer to the non-versioned TR url for each reference. We have run into the following issues with the Bikeshed biblio reference tags.
[UAAG]
A non-versioned url exists, however, it opens the 1.0 spec, https://www.w3.org/TR/UAAG/ Spec currently uses[UAAG20] tag for the informative reference which links to the 2.0 spec, https://www.w3.org/TR/UAAG20.
[CSS] A a non-versioned url is available, https://www.w3.org/TR/CSS/. However, this receives an error from Bikeshed. Need to use a specific version, e.g. [CSS2].
[HTTP] Spec uses [HTTP11]. I don't believe there is a non-versioned url.
[SVG] There is a non-versioned url, however, it opens the 1.1 spec, https://www.w3.org/TR/SVG/. Spec uses [SVG2], https://www.w3.org/TR/SVG2/.
[WAI-ARIA] References WAI-ARIA 1.0. Should reference non-versioned TR, https://www.w3.org/TR/wai-aria/, which opens WAI-ARIA 1.1

**Comment by moekraft** [comment]
@nitedog Shadi, I opened this Bikeshed issue to address ACT Rules Issue 216: https://github.com/w3c/wcag-act/issues/216

**Comment by tabatkins** [comment]
So, unfortunately, I can't do a lot about this. My bibliography references come from [SpecRef](https://www.specref.org/), and all of SpecRef's W3C references come from the automated W3C feed. You can add explicit corrections to SpecRef individually, but usually the best idea is to contact PLH explicitly and try to get these unversioned references added/fixed.

(It's strange that CSS doesn't exist, given that we *do* have `/TR/CSS` pointing to the snapshot; that should be fixed. UUAG, SVG, and WAI-ARIA definitely should all be fixed so that the unversioned points to latest. Dunno what org maintains the HTTP11 reference; it's just an alias for RFC7230.)

**Comment by nitedog** [comment]
I'm on it

**Comment by plehegar** [comment]
We deployed different flavors for folks last year:
 https://www.w3.org/2005/05/tr-versions#shortnames

eg for SVG, we have:
 https://www.w3.org/TR/SVG/
 https://www.w3.org/TR/SVG/latest/
 https://www.w3.org/TR/SVG/upcoming/

@deniak , that doesn't seem to work with CSS however...

**Comment by deniak** [comment]
CSS is a special use case as it split into different modules, so we can't point to a specific version unless we decide we want to redirect to older version but I don't think it's an option.

**Comment by plehegar** [comment]
But CSS is about the snapshot nowadays...

**Comment by tabatkins** [comment]
Yeah, `/TR/CSS` points to the snapshot; `[CSS]` in a spec should point to the snapshot too, ideally.

**Comment by plehegar** [comment]
I started to talk to Denis about this. Don't expect a fast turnaround unfortunately since the bug is pretty fundamental imho and will require substantive work to fix this properly our API to Do the Right Thing. We simply don't have the information stored in a way that is currently exposed to the API.

**Comment by nitedog** [comment]
@plehegar & @tabatkins can you please make the following changes for references:

- WCAG: should refer to https://www.w3.org/TR/WCAG/ (currently v2.1) rather than https://www.w3.org/TR/WAI-WEBCONTENT/ (refers to v1.0)
- WAI-ARIA: should now refer to v1.1 (https://www.w3.org/TR/wai-aria/ is correct)

Thanks!
